awx/main/models/label.py

```diff
@@ -10,6 +10,8 @@
 from awx.main.models.base import CommonModelNameNotUnique
 from awx.main.models.unified_jobs import UnifiedJobTemplate, UnifiedJob
 from awx.main.models.inventory import Inventory
+from awx.main.models.schedules import Schedule
+from awx.main.models.workflow import WorkflowJobTemplateNode, WorkflowJobNode
```

Comment:
It always worries me to add more imports from these kind of sibling modules, because there's not a clear hierarchy / ordering of the models, and can lead to circular imports... or just make it more easy to create circular imports in the future.

Reply:
100%, I tired to do a more generic import and got a circular reference.